### PR TITLE
Added support for refreshing currencies from MNB homepage

### PR DESCRIPTION
**Pull request by gergob**
Please let me know if you think I should change something in the implementation.

Thanks,
Greg

### CODE REVIEW

File: currency_rate_update/model/company.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def _compute_multi_curr_enable(self):
     @api.one
     def button_refresh_currency(self):
         """Refresh the currencies rates !!for all companies now"""
+        print self.services_to_use
```
Comment (nagyv) [inline currency_rate_update/model/company.py]: please remove this

File: currency_rate_update/model/company.py [hunk 2]
```diff
@@ -56,5 +57,5 @@ def button_refresh_currency(self):
     # List of services to fetch rates
     services_to_use = fields.One2many(
         'currency.rate.update.service',
-        'company_id',
+        'company_id', #company_id
```
Comment (nagyv) [inline currency_rate_update/model/company.py]: I guess, this remained, but is not needed. As I would like the pull request to be accepted in the original OCA repo, we need as few changes as necessary.

File: currency_rate_update/model/currency_rate_update.py [hunk 3]
```diff
@@ -237,19 +247,23 @@ def _onchange_service(self):
     @api.one
     def refresh_currency(self):
         """Refresh the currencies rates !!for all companies now"""
+        _logger.info("===>CURR: Refreshing currency")
```
Comment (nagyv) [inline currency_rate_update/model/currency_rate_update.py]: Please, remove the logger lines, and unnecessary comments.

File: currency_rate_update/model/currency_rate_update.py [hunk 4]
```diff
@@ -313,6 +331,9 @@ def refresh_currency(self):
                             _intervalTypes[str(self.interval_type)]
                             (self.interval_number)).date()
                 self.next_run = next_run
+        else:
+            raise exceptions.Warning(_('If "Auto update" is not set, currency refresh will not work.'))
```
Comment (nagyv) [inline currency_rate_update/model/currency_rate_update.py:334]: Great idea for quick handling of this stupid case. Thank you!

File: currency_rate_update/services/__init__.py [hunk 5]
```diff
@@ -18,3 +18,4 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+from . import update_service_HU_MNB
```
Comment (nagyv) [inline currency_rate_update/services/__init__.py]: Are you sure that this is necessary? The other services seemingly don't need to be imported here.

File: currency_rate_update/services/update_service_HU_MNB.py [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+_logger = logging.getLogger(__name__)
+
+
+class HU_MNB_getter(Currency_getter_interface):
+    """Implementation of Currency_getter_factory interface
+    for the Hungarian National Bank (http://www.mnb.hu)
+    """
+
+    def get_updated_currency(self, currency_array, main_currency,
+                             max_delta_days):
```
Comment (nagyv) [inline currency_rate_update/services/update_service_HU_MNB.py:79]: max_delta_days is unused, even though the last date of refresh is available in the table of currencies, please add support for max_delta_days as well (only currencies fresher than today-max_delta_days should be registered)

File: currency_rate_update/services/__init__.py (hunk 5 above)
Comment (gergob) [inline currency_rate_update/services/__init__.py]: No I am not, I'll remove it.

File: currency_rate_update/services/update_service_HU_MNB.py (hunk 6 above)
Comment (gergob) [inline currency_rate_update/services/update_service_HU_MNB.py:79]: OK, I missed that part, sending a new pull request soon.

**Comment by gergob** [comment]
Added the requested changes, please let me know if you'd like to change anything else.

**Pull request merged**